openreview/conference/helpers.py

```diff
@@ -125,6 +125,11 @@ def get_conference_builder(client, request_form_id, support_user='OpenReview.net
 
     name = note.content.get('submission_name', 'Submission').strip()
 
+    readers = None
+
+    if 'Reviewer Bid Scores' in note.content.get('Paper Matching', '') or 'Reviewer Recommendation Scores' in note.content.get('Paper Matching', ''):
+        readers = [openreview.SubmissionStage.Readers.SENIOR_AREA_CHAIRS, openreview.SubmissionStage.Readers.AREA_CHAIRS, openreview.SubmissionStage.Readers.REVIEWERS]
```

Comment:
I wonder if we should allow the PCs to define the readers in the "Post Submission Stage" instead of us deciding them. 

I'm saying this because in the future I would like to remoce the field "Paper Matching" from the request form and create BidStage always. The PCs can decide if they want to use it or not.

Reply:
The only problem I see with this is that not all PCs use the 'Post Submission' button, I think most of the times `setup_post_submission_stage()` is being called from the process fucntion of one of the stages, should we remove it and force PCs to use the button?